packages/expo-yarn-workspaces/bin/symlink-necessary-packages.js

```diff
@@ -16,6 +16,12 @@ function symlinkNecessaryPackages(projectPath) {
 
   symlinkNecessaryPackage(projectPath, 'expo');
   symlinkNecessaryPackage(projectPath, 'react-native');
+  // For community CLI autolinking in bare-expo
+  symlinkNecessaryPackage(projectPath, 'expo-image');
+  symlinkNecessaryPackage(projectPath, 'expo-random');
+  symlinkNecessaryPackage(projectPath, 'expo-dev-launcher');
+  symlinkNecessaryPackage(projectPath, 'expo-dev-menu');
+  symlinkNecessaryPackage(projectPath, 'expo-dev-menu-interface');
```

Comment:
I'm not sure if we need to create symlinks for those packages. RN CLI link should work without that. Or maybe there is another reason why you add that?

Reply:
afaict without these symlinks, the packages don't get autolinked to bare-expo. Can test by removing, running yarn, and running pod install in `apps/bare-expo/ios`. The packages will then be uninstalled.

Reply:
(a) keep code specific to bare-expo out of expo-yarn-workspaces. can put it in bare-expo instead.
(b) what does the autolinking code in the RN CLI do? does it only look in the project root's node_modules?

Reply:
Note that the `expo-dev-menu-interface` isn't listed in the podfile now, but it still is linked into the project (without the symlink). 

Moreover, the RN config lists the package correctly on both platforms: 
```
    "expo-dev-menu-interface": {
      "root": "/Users/lukasz/work/expo/packages/expo-dev-menu-interface",
      "name": "expo-dev-menu-interface",
      "platforms": {
        "ios": {
          "sourceDir": "/Users/lukasz/work/expo/packages/expo-dev-menu-interface/ios/EXDevMenuInterface.xcodeproj",
          "folder": "/Users/lukasz/work/expo/packages/expo-dev-menu-interface",
          "pbxprojPath": "/Users/lukasz/work/expo/packages/expo-dev-menu-interface/ios/EXDevMenuInterface.xcodeproj/project.pbxproj/project.pbxproj",
          "podfile": null,
          "podspecPath": "/Users/lukasz/work/expo/packages/expo-dev-menu-interface/EXDevMenuInterface.podspec",
          "projectPath": "/Users/lukasz/work/expo/packages/expo-dev-menu-interface/ios/EXDevMenuInterface.xcodeproj/project.pbxproj",
          "projectName": "project.pbxproj",
          "libraryFolder": "Libraries",
          "sharedLibraries": [],
          "plist": [],
          "scriptPhases": []
        },
        "android": {
          "sourceDir": "/Users/lukasz/work/expo/packages/expo-dev-menu-interface/android",
          "folder": "/Users/lukasz/work/expo/packages/expo-dev-menu-interface",
          "packageImportPath": "import expo.interfaces.devmenu.DevMenuInterfacePackage;",
          "packageInstance": "new DevMenuInterfacePackage()"
        }
      },
      "assets": [],
      "hooks": {},
      "params": []
    },
```

Reply:
but maybe after you merged https://github.com/expo/expo/pull/11402, it stopped working. If that's the case, I'm ok with these changes ;)